CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ https://github.com/bitcrowd/rubocop-bitcrowd/compare/v2.2.0...HEAD
 
 ### Fixes:
 
-* *Put fixes here (in a brief bullet point)*
+* [#42](https://github.com/bitcrowd/rubocop-bitcrowd/pull/42) Remove deprecation warnings due to `ExcludedMethods` in rubocop >= 1.5.0
```

Comment:
If this change only works with Rubocop versions >= 1.5.0, I'd say we should also update / change  the version this gem requires to work with:

https://github.com/bitcrowd/rubocop-bitcrowd/blob/e0ec2fe1e1cd864bb5c9a9782bbe8ca12b13e051/rubocop-bitcrowd.gemspec#L53

How about:
```
spec.add_runtime_dependency 'rubocop', '>= 1.5.0'
```

This will make updates on projects easier to handle. I personally also don't really see the point in trying to be backwards compatible for ages with our gem 🤷